### DISCUSSION TITLE
SDXL inpaint

### DISCUSSION THREAD

**Discussion by kalle07** [hako-mikan/sd-webui-supermerger#430] [Announcements]
hey 

iam not shure but normal SDXL model are able to inpaint but i think not as good as inpaintmodels from 1.5

so it exist an inpaint model SDXL
https://civitai.com/models/146028/sdxl-inpainting-01-official-reupload
but its half precision (16GB Vram cant handle) you can recalculate this ? 

so if i use the procedure of creating an inpaint model
https://civitai.com/articles/1045/inpainting-for-any-model

its works so fare without error but it does not inpainting, allways a grey area ...
the build in checkpointmerger in 1111 works different, this model also create a model and save but if i start it runs out of memory like the original inpaint_model ...

**Comment by hako-mikan** [comment]
Can't quite grasp the situation, but if out of memory is the issue, then shouldn't you first save each model in fp16, and then create the inpaint model using the method you mentioned?

**Comment by kalle07** [comment]
i doo, i saved original SDXL inpaint model in fp16 but after that it doesnt work anymore ...
it is half precision but its only 6GB  ?!? i dont realy understand ^^

my intention is to make a real inpaint-model like in SD15 ...
or you have a better idea  ?  :)
